src/methods/modular_programs/easy/q5/HeightInFeet.java

```diff
@@ -0,0 +1,30 @@
+package methods.modular_programs.easy.q5;
+
+/*
+Write a Java Program that asks users for their height in cm, then convert that height in feet. Write a method to convert
```

Comment:
Change question text to: 
`
Write a program that asks users for their height in cm, then converts it to feet. Create a method called heightConversion to do this, which should return the value in feet, or -1 of the entered height is less than zero. 
`